var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -141,6 +141,9 @@ class FluxCore(AutotoolsPackage):
     # `link` dependency on python due to Flux's `pymod` module
     depends_on("python@3.6:", when="@0.17:", type=("build", "link", "run"))
     depends_on("python@2.7:", type=("build", "link", "run"))
+    # See https://github.com/flux-framework/flux-core/issues/5455 for
+    # Python 3.12 distutils issues
+    depends_on("python@:3.11", type=("build", "link", "run"))
```

Comment:
This issue has been fixed in flux-core@master, so we should specify this constraint only for flux-core <= 0..54 if possible.

Reply:
Thanks